### PR TITLE
fix: update default alert ch queries for traces and logs

### PR DESCRIPTION
**Pull request by nityanandagohain**
Part of https://github.com/SigNoz/engineering-pod/issues/2036

* Added ts_bucket_start filter to both the queries.
* for traces update the table name the attributes key

Also @srikanthccv  is peer.service is something we want to keep in this query ?

----

> [!IMPORTANT]
> Update default alert queries in `defaults.ts` by adding `ts_bucket_start` filter and modifying trace query table name and attribute key.
> 
>   - **Behavior**:
>     - Add `ts_bucket_start` filter to log and trace queries in `defaults.ts`.
>     - Update trace query table name to `signoz_traces.distributed_signoz_index_v3` and attribute key to `resource_string_service$$name`.
>   - **Misc**:
>     - Update query in `logAlertDefaults` to include `ts_bucket_start` filter.
>     - Update query in `traceAlertDefaults` to include `ts_bucket_start` filter and change table name and attribute key.
> 
> <sup>This description was created by </sup>[Ellipsis](https://www.ellipsis.dev?ref=SigNoz%2Fsignoz&utm_source=github&utm_medium=referral)<sup> for 9f2d97137d45eeffee0feafbc539048f10be0616. It will automatically update as commits are pushed.</sup>

### CODE REVIEW

**Comment by srikanthccv** [comment]
let's change it to use service.name

File: frontend/src/container/CreateAlertRule/defaults.ts [hunk 1]
```diff
@@ -149,7 +149,7 @@ export const traceAlertDefaults: AlertDef = {
 			chQueries: {
 				A: {
 					name: 'A',
-					query: `SELECT \n\ttoStartOfInterval(timestamp, INTERVAL 1 MINUTE) AS interval, \n\tstringTagMap['peer.service'] AS op_name, \n\ttoFloat64(avg(durationNano)) AS value \nFROM signoz_traces.distributed_signoz_index_v2  \nWHERE stringTagMap['peer.service']!='' \nAND timestamp BETWEEN {{.start_datetime}} AND {{.end_datetime}} \nGROUP BY (op_name, interval);\n\n-- available variables:\n-- \t{{.start_datetime}}\n-- \t{{.end_datetime}}\n\n-- required column alias:\n-- \tvalue\n-- \tinterval`,
+					query: `SELECT \n\ttoStartOfInterval(timestamp, INTERVAL 1 MINUTE) AS interval, \n\tresource_string_service$$name AS op_name, \n\ttoFloat64(avg(duration_nano)) AS value \nFROM signoz_traces.distributed_signoz_index_v3  \nWHERE resource_string_service$$name !='' \nAND timestamp BETWEEN {{.start_datetime}} AND {{.end_datetime}} \nAND ts_bucket_start BETWEEN {{.start_timestamp}} - 1800 AND {{.end_timestamp}} \nGROUP BY (op_name, interval);\n\n-- available variables:\n-- \t{{.start_datetime}}\n-- \t{{.end_datetime}}\n-- \t{{.start_timestamp}}\n-- \t{{.end_timestamp}}\n\n-- required column alias:\n-- \tvalue\n-- \tinterval`,
```
Comment (srikanthccv) [inline frontend/src/container/CreateAlertRule/defaults.ts:152]:
```suggestion
					query: `SELECT \n\ttoStartOfInterval(timestamp, INTERVAL 1 MINUTE) AS interval, \n\tresource_string_service$$name AS `service.name`, \n\ttoFloat64(avg(duration_nano)) AS value \nFROM signoz_traces.distributed_signoz_index_v3  \nWHERE resource_string_service$$name !='' \nAND timestamp BETWEEN {{.start_datetime}} AND {{.end_datetime}} \nAND ts_bucket_start BETWEEN {{.start_timestamp}} - 1800 AND {{.end_timestamp}} \nGROUP BY (`service.name`, interval);\n\n-- available variables:\n-- \t{{.start_datetime}}\n-- \t{{.end_datetime}}\n-- \t{{.start_timestamp}}\n-- \t{{.end_timestamp}}\n\n-- required column alias:\n-- \tvalue\n-- \tinterval`,
```

**Review by srikanthccv — COMMENTED** [review]
Do we have docs somewhere explaining the bucket timestamp? If not, please create them and link the docs in the query comments

**Comment by nityanandagohain** [comment]
Will have to update the docs for clickhouse queries all together which is a sepearete task, will do that and get back to this. Will hold this pr till then.

**Review by YounixM — APPROVED** [review]

**Pull request merged**